climakitae/util/utils.py

```diff
@@ -520,3 +520,40 @@ def summary_table(data):
         df = df.sort_values(by=["year"])
 
     return df
+
+
+def convert_to_local_time(data, selections, lat, lon) -> xr.Dataset:
```

Comment:
I'm not seeing where lat and lon are actually used in this function. Presumably this information is needed to determine the local timezone, but at present not being used because the timezone is hardcoded to PST

Reply:
Yep! Fixing this later today.